### PR TITLE
Add PaladiN and oldschool

### LINKED ISSUE #50: WIP Skins: PaladiN and Oldschool
Im considering to work on these skins, so if you will work on these, you may prefer to select a different graphic or post a comment in this issue.

_(I dont guarantee finishing them)_

### PR DESCRIPTION
**Pull request by M0REKZ**
SVGs for PaladiN and oldschool

I took the Whis signature from the svgs made by mind, however, i couldn't fit very well all those words at the oldschool skin
:(.

closes #50

### CODE REVIEW

**Comment by ChillerDragon** [comment]
Nice thanks @M0REKZ ! :rocket: 

Did not take a close look but LGTM so far.

**Review by ChillerDragon — APPROVED** [review]

File: skins/PaladiN.svg
Comment (Jupeyy) [inline skins/PaladiN.svg:1]: mh you put it over one of the inactive eyes

@ChillerDragon opinion on that? will it maybe be used in future?

Comment (M0REKZ) [reply skins/PaladiN.svg:1]: On the past pull request i made that on beast too

Comment (ChillerDragon) [reply skins/PaladiN.svg:1]: Oh yea thats a proper black on black hack :D
But fine with me. If it starts to bug we can fix it.

**Pull request merged**